indexer-monitor/README.md

```diff
@@ -6,3 +6,12 @@ Checks whether the indexer is working properly.
 * (info level) Sends a report when the indexer's synchronization status returns to normal.
 * (info level) Sends a report when the CodeChain node that the indexer is watching is normal.
 * (info level) Sends a report that notifies whether the indexer is running once a day.
+
+| Key               | Description                                 | Example values              | Default value |
+| ----------------- | ------------------------------------------- | --------------------------- | ------------- |
+| NODE_ENV          | Use "production" to reduce mistakes         | production OR test          | null          |
+| NETWORK_ID        | Network ID of connected CodeChain           | cc, bc, tc                  | null          |
```

Comment:
of connected CodeChain node? Maybe should be more specific here?

Reply:
Good. "connected CodeChan" is obscure.